build-logic/project-plugins/src/main/kotlin/com.hedera.gradle.services.gradle.kts

```diff
@@ -20,3 +20,11 @@ plugins {
 }
 
 group = "com.hedera.hashgraph"
+
+// TODO jjohannes: remove and fix issues in services code base
```

Comment:
To reduce the numbers of required code owner reviews for this one, I will do the Javadoc fixes in the services code base as a follow up.